### PR TITLE
Enable us_ticker tests

### PR DESCRIPTION
**Pull request by fkjagodzinski**
This patch depends on https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6.

1. Enable **us_ticker** greentea tests:
    - `us_ticker`,
    - `us_ticker_lp_ticker_common`, previously named `common_tickers`,
    - `us_ticker_lp_ticker_frequency`, previously named `common_tickers_freq`.

1. Fix test README.
1. Disable the `LPTICKER` (out of scope currently).
1. Add a few Bash scripts for building and running tests.

### CODE REVIEW

**Review by LMESTM — DISMISSED** [review]
I was about to request change of the git URL, but this is done now so Looks good to me !

**Comment by fkjagodzinski** [comment]
Thanks for a review @LMESTM! I forgot to update the actual MCU-Driver-HAL submodule URL to https. This is fixed now.

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "MCU-Driver-HAL"]
 	path = MCU-Driver-HAL
-	url = git@github.com:MCU-Driver-HAL/MCU-Driver-HAL.git
+	url = https://github.com/MCU-Driver-HAL/MCU-Driver-HAL.git
```
Comment (MarceloSalazar) [inline .gitmodules:3]:
```suggestion
	url = https://github.com/MCU-Driver-HAL/MCU-Driver-HAL
```

Comment (MarceloSalazar) [reply .gitmodules:3]: @fkjagodzinski can you rebase?
This should have been addressed in https://github.com/MCU-Driver-HAL/MCU-Driver-ST/pull/4

**Comment by fkjagodzinski** [comment]
I rebased the commits and made a couple of updates to match the current state of https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6, for details see https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6#issuecomment-868468765.

**Comment by fkjagodzinski** [comment]
Converting this to a draft PR for now; this has to wait for https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6 to be merged first.

**Comment by fkjagodzinski** [comment]
Yet another update to keep up with https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6#issuecomment-869806253. Additionally, fixed the Bash scripts.

**Comment by fkjagodzinski** [comment]
Since https://github.com/MCU-Driver-HAL/MCU-Driver-HAL/pull/6 is merged now, this one is ready to go.

File: sdfx_st/tests/build_all.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TESTS_BUILT=()
+
+for TEST_PATH in $(find ./ -type f -name CMakeLists.txt); do
+    TEST_DIR_PATH=$(dirname "$TEST_PATH")
+    printf 'Building %s\n' "$TEST_DIR_PATH"
```
[the remark this answers is not in the document]
Comment (MarceloSalazar) [reply sdfx_st/tests/build_all.sh:22]: Not a problem for now

**Comment by MarceloSalazar** [comment]
@fkjagodzinski looks good.
Can you please attach a summary of us_Ticker Greentea tests report to this PR as future reference?

**Comment by fkjagodzinski** [comment]
Here is the output of the `run_all.sh` script executed from `MCU-Driver-ST/sdfx_st/tests` dir:

<details>
<summary>See the greentea output.</summary>

```
Running ./mbed_hal/us_ticker/cmake_build/sdfx-st-test-us_ticker.bin
[1625054645.64][HTST][INF] host test executor ver. 0.0.15
[1625054645.64][HTST][INF] copy image onto target...
[1625054650.75][MBED][WRN] Target ID not found: Skipping flash check and retry
[1625054650.76][HTST][INF] starting host test process...
[1625054650.76][CONN][INF] starting connection process...
[1625054650.76][CONN][INF] notify event queue about extra 60 sec timeout for serial port pooling
[1625054650.76][CONN][INF] initializing serial port listener... 
[1625054650.76][SERI][INF] serial(port=/dev/ttyACM0, baudrate=115200, read_timeout=0.01, write_timeout=5)
[1625054650.76][HTST][INF] setting timeout to: 60 sec
[1625054650.76][SERI][INF] reset device using 'default' plugin...
[1625054650.87][SERI][INF] waiting 1.00 sec after reset
[1625054651.87][SERI][INF] wait for it...
[1625054651.87][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
[1625054651.87][CONN][INF] sending up to 2 __sync packets (specified with --sync=2)
[1625054651.87][CONN][INF] sending preamble '0115f461-77c4-4fd0-99ad-d35cbd34c121'
[1625054651.88][SERI][TXD] {{__sync;0115f461-77c4-4fd0-99ad-d35cbd34c121}}
[1625054651.89][CONN][RXD] mbedmbedmbedmbedmbedmbedmbedmbed
[1625054651.89][CONN][INF] found SYNC in stream: {{__sync;0115f461-77c4-4fd0-99ad-d35cbd34c121}} it is #0 sent, queued...
[1625054651.89][HTST][INF] sync KV found, uuid=0115f461-77c4-4fd0-99ad-d35cbd34c121, timestamp=1625054651.886324
[1625054651.90][CONN][RXD] >>> Running 1 test cases...
[1625054651.90][CONN][INF] found KV pair in stream: {{__version;1.3.0}}, queued...
[1625054651.90][CONN][INF] found KV pair in stream: {{__timeout;20}}, queued...
[1625054651.90][CONN][INF] found KV pair in stream: {{__host_test_name;default_auto}}, queued...
[1625054651.90][CONN][INF] found KV pair in stream: {{__testcase_name;us ticker info test}}, queued...
[1625054651.90][HTST][INF] DUT greentea-client version: 1.3.0
[1625054651.90][HTST][INF] setting timeout to: 20 sec
[1625054651.90][HTST][INF] host test class: '<class 'mbed_os_tools.test.host_tests.default_auto.DefaultAuto'>'
[1625054651.90][HTST][INF] host test setup() call...
[1625054651.90][HTST][INF] CALLBACKs updated
[1625054651.90][HTST][INF] host test detected: default_auto
[1625054651.91][CONN][RXD] 
[1625054651.91][CONN][RXD] >>> Running case #1: 'us ticker info test'...
[1625054651.91][CONN][INF] found KV pair in stream: {{__testcase_start;us ticker info test}}, queued...
[1625054651.92][CONN][RXD] >>> 'us ticker info test': 1 passed, 0 failed
[1625054651.92][CONN][RXD] 
[1625054651.92][CONN][RXD] >>> Test cases: 1 passed, 0 failed
[1625054651.92][CONN][INF] found KV pair in stream: {{__testcase_finish;us ticker info test;1;0}}, queued...
[1625054651.93][CONN][INF] found KV pair in stream: {{__testcase_summary;1;0}}, queued...
[1625054651.93][CONN][INF] found KV pair in stream: {{end;success}}, queued...
[1625054651.93][CONN][INF] found KV pair in stream: {{__exit;0}}, queued...
[1625054651.93][HTST][INF] __exit(0)
[1625054651.93][HTST][INF] __notify_complete(True)
[1625054651.93][HTST][INF] __exit_event_queue received
[1625054651.93][HTST][INF] test suite run finished after 0.03 sec...
[1625054651.94][CONN][INF] received special event '__host_test_finished' value='True', finishing
[1625054651.94][HTST][INF] CONN exited with code: 0
[1625054651.94][HTST][INF] Some events in queue
[1625054651.94][HTST][INF] stopped consuming events
[1625054651.94][HTST][INF] host test result() call skipped, received: True
[1625054651.94][HTST][INF] calling blocking teardown()
[1625054651.94][HTST][INF] teardown() finished
[1625054651.94][HTST][INF] {{result;success}}
Running ./mbed_hal/us_ticker_lp_ticker_frequency/cmake_build/sdfx-st-test-us_ticker_lp_ticker_frequency.bin
[1625054652.31][HTST][INF] host test executor ver. 0.0.15
[1625054652.31][HTST][INF] copy image onto target...
[1625054657.45][MBED][WRN] Target ID not found: Skipping flash check and retry
[1625054657.45][HTST][INF] starting host test process...
[1625054657.45][CONN][INF] starting connection process...
[1625054657.45][CONN][INF] notify event queue about extra 60 sec timeout for serial port pooling
[1625054657.46][CONN][INF] initializing serial port listener... 
[1625054657.46][HTST][INF] setting timeout to: 60 sec
[1625054657.46][SERI][INF] serial(port=/dev/ttyACM0, baudrate=115200, read_timeout=0.01, write_timeout=5)
[1625054657.46][SERI][INF] reset device using 'default' plugin...
[1625054657.57][SERI][INF] waiting 1.00 sec after reset
[1625054658.57][SERI][INF] wait for it...
[1625054658.57][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
[1625054658.57][CONN][INF] sending up to 2 __sync packets (specified with --sync=2)
[1625054658.57][CONN][INF] sending preamble '7eb8908a-cd54-4f19-89bd-b41c2e3af33a'
[1625054658.58][SERI][TXD] {{__sync;7eb8908a-cd54-4f19-89bd-b41c2e3af33a}}
[1625054658.59][CONN][RXD] mbedmbedmbedmbedmbedmbedmbedmbed
[1625054658.59][CONN][INF] found SYNC in stream: {{__sync;7eb8908a-cd54-4f19-89bd-b41c2e3af33a}} it is #0 sent, queued...
[1625054658.59][HTST][INF] sync KV found, uuid=7eb8908a-cd54-4f19-89bd-b41c2e3af33a, timestamp=1625054658.587039
[1625054658.60][CONN][RXD] >>> Running 1 test cases...
[1625054658.60][CONN][INF] found KV pair in stream: {{__version;1.3.0}}, queued...
[1625054658.60][CONN][INF] found KV pair in stream: {{__timeout;120}}, queued...
[1625054658.60][CONN][INF] found KV pair in stream: {{__host_test_name;timing_drift_auto}}, queued...
[1625054658.60][CONN][INF] found KV pair in stream: {{__testcase_count;1}}, queued...
[1625054658.60][HTST][INF] DUT greentea-client version: 1.3.0
[1625054658.60][HTST][INF] setting timeout to: 120 sec
[1625054658.60][HTST][INF] host test class: '<class 'timing_drift_auto.TimingDriftSync'>'
[1625054658.60][HTST][INF] host test setup() call...
[1625054658.60][HTST][INF] CALLBACKs updated
[1625054658.60][HTST][INF] host test detected: timing_drift_auto
[1625054658.61][CONN][RXD] 
[1625054658.61][CONN][RXD] >>> Running case #1: 'Microsecond ticker frequency test'...
[1625054658.61][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker frequency test}}, queued...
[1625054658.62][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker frequency test}}, queued...
[1625054658.62][CONN][INF] found KV pair in stream: {{timing_drift_check_start;0}}, queued...
[1625054658.63][SERI][TXD] {{base_time;0}}
[1625054658.64][CONN][INF] found KV pair in stream: {{base_time;23195}}, queued...
[1625054665.41][HTST][INF] Device base time 23195
[1625054665.41][HTST][INF] sleeping for 6.76177978515625 to measure drift accurately
[1625054665.42][SERI][TXD] {{final_time;0}}
[1625054665.43][CONN][INF] found KV pair in stream: {{final_time;6807197}}, queued...
[1625054665.43][HTST][INF] Device final time 6807197 
[1625054665.43][HTST][INF] Compute host events
[1625054665.43][HTST][INF] Transport delay 0: 0.022022247314453125
[1625054665.43][HTST][INF] Transport delay 1: 0.020592451095581055
[1625054665.43][HTST][INF] DUT base time : 23195.0
[1625054665.43][HTST][INF] DUT end time  : 6807197.0
[1625054665.43][HTST][INF] min_pass : 6.4686684727668755 , max_pass : 7.104835510253906 for 5.0%%
[1625054665.43][HTST][INF] min_inconclusive : 6.428184509277344 , max_inconclusive : 7.149580943584443
[1625054665.43][HTST][INF] Time reported by device: 6.784002
[1625054665.43][HTST][INF] Test passed !!!
[1625054665.44][SERI][TXD] {{pass;0}}
[1625054665.45][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker frequency test;1;0}}, queued...
[1625054665.46][CONN][RXD] >>> 'Microsecond ticker frequency test': 1 passed, 0 failed
[1625054665.46][CONN][RXD] 
[1625054665.46][CONN][RXD] >>> Test cases: 1 passed, 0 failed
[1625054665.46][CONN][INF] found KV pair in stream: {{__testcase_summary;1;0}}, queued...
[1625054665.46][CONN][INF] found KV pair in stream: {{end;success}}, queued...
[1625054665.46][HTST][INF] __notify_complete(True)
[1625054665.46][HTST][INF] __exit_event_queue received
[1625054665.46][HTST][INF] test suite run finished after 6.86 sec...
[1625054665.47][CONN][INF] found KV pair in stream: {{__exit;0}}, queued...
[1625054665.47][CONN][INF] received special event '__host_test_finished' value='True', finishing
[1625054665.48][HTST][INF] CONN exited with code: 0
[1625054665.48][HTST][INF] Some events in queue
[1625054665.48][HTST][INF] stopped consuming events
[1625054665.48][HTST][INF] host test result() call skipped, received: True
[1625054665.48][HTST][WRN] missing __exit event from DUT
[1625054665.48][HTST][INF] calling blocking teardown()
[1625054665.48][HTST][INF] teardown() finished
[1625054665.48][HTST][INF] {{result;success}}
Running ./mbed_hal/us_ticker_lp_ticker_common/cmake_build/sdfx-st-test-us_ticker_lp_ticker_common.bin
[1625054665.84][HTST][INF] host test executor ver. 0.0.15
[1625054665.84][HTST][INF] copy image onto target...
[1625054671.12][MBED][WRN] Target ID not found: Skipping flash check and retry
[1625054671.13][HTST][INF] starting host test process...
[1625054671.14][CONN][INF] starting connection process...
[1625054671.14][CONN][INF] notify event queue about extra 60 sec timeout for serial port pooling
[1625054671.14][CONN][INF] initializing serial port listener... 
[1625054671.14][SERI][INF] serial(port=/dev/ttyACM0, baudrate=115200, read_timeout=0.01, write_timeout=5)
[1625054671.14][HTST][INF] setting timeout to: 60 sec
[1625054671.14][SERI][INF] reset device using 'default' plugin...
[1625054671.25][SERI][INF] waiting 1.00 sec after reset
[1625054672.25][SERI][INF] wait for it...
[1625054672.25][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
[1625054672.25][CONN][INF] sending up to 2 __sync packets (specified with --sync=2)
[1625054672.25][CONN][INF] sending preamble '39c2c515-7fc7-4228-b6a0-0edb73415d15'
[1625054672.26][SERI][TXD] {{__sync;39c2c515-7fc7-4228-b6a0-0edb73415d15}}
[1625054672.27][CONN][RXD] mbedmbedmbedmbedmbedmbedmbedmbed
[1625054672.27][CONN][INF] found SYNC in stream: {{__sync;39c2c515-7fc7-4228-b6a0-0edb73415d15}} it is #0 sent, queued...
[1625054672.27][CONN][INF] found KV pair in stream: {{__version;1.3.0}}, queued...
[1625054672.27][HTST][INF] sync KV found, uuid=39c2c515-7fc7-4228-b6a0-0edb73415d15, timestamp=1625054672.267307
[1625054672.27][HTST][INF] DUT greentea-client version: 1.3.0
[1625054672.28][CONN][RXD] >>> Running 9 test cases...
[1625054672.28][CONN][INF] found KV pair in stream: {{__timeout;80}}, queued...
[1625054672.28][CONN][INF] found KV pair in stream: {{__host_test_name;default_auto}}, queued...
[1625054672.28][HTST][INF] setting timeout to: 80 sec
[1625054672.28][HTST][INF] host test class: '<class 'mbed_os_tools.test.host_tests.default_auto.DefaultAuto'>'
[1625054672.28][HTST][INF] host test setup() call...
[1625054672.28][HTST][INF] CALLBACKs updated
[1625054672.28][HTST][INF] host test detected: default_auto
[1625054672.29][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker init is safe to call repeatedly}}, queued...
[1625054672.29][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker info test}}, queued...
[1625054672.30][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker interrupt test}}, queued...
[1625054672.30][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker past interrupt test}}, queued...
[1625054672.30][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker reschedule test}}, queued...
[1625054672.31][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker fire interrupt}}, queued...
[1625054672.31][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker overflow test}}, queued...
[1625054672.32][CONN][RXD] 
[1625054672.32][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker increment test}}, queued...
[1625054672.32][CONN][INF] found KV pair in stream: {{__testcase_name;Microsecond ticker speed test}}, queued...
[1625054672.33][CONN][RXD] >>> Running case #1: 'Microsecond ticker init is safe to call repeatedly'...
[1625054672.33][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker init is safe to call repeatedly}}, queued...
[1625054672.34][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker init is safe to call repeatedly;1;0}}, queued...
[1625054672.36][CONN][RXD] >>> 'Microsecond ticker init is safe to call repeatedly': 1 passed, 0 failed
[1625054672.36][CONN][RXD] 
[1625054672.36][CONN][RXD] >>> Running case #2: 'Microsecond ticker info test'...
[1625054672.36][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker info test}}, queued...
[1625054672.37][CONN][RXD] >>> 'Microsecond ticker info test': 1 passed, 0 failed
[1625054672.37][CONN][RXD] 
[1625054672.37][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker info test;1;0}}, queued...
[1625054672.38][CONN][RXD] >>> Running case #3: 'Microsecond ticker interrupt test'...
[1625054672.38][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker interrupt test}}, queued...
[1625054672.40][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker interrupt test;1;0}}, queued...
[1625054672.41][CONN][RXD] >>> 'Microsecond ticker interrupt test': 1 passed, 0 failed
[1625054672.41][CONN][RXD] 
[1625054672.41][CONN][RXD] >>> Running case #4: 'Microsecond ticker past interrupt test'...
[1625054672.42][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker past interrupt test}}, queued...
[1625054672.42][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker past interrupt test;1;0}}, queued...
[1625054672.43][CONN][RXD] >>> 'Microsecond ticker past interrupt test': 1 passed, 0 failed
[1625054672.43][CONN][RXD] 
[1625054672.43][CONN][RXD] >>> Running case #5: 'Microsecond ticker reschedule test'...
[1625054672.44][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker reschedule test}}, queued...
[1625054672.44][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker reschedule test;1;0}}, queued...
[1625054672.46][CONN][RXD] >>> 'Microsecond ticker reschedule test': 1 passed, 0 failed
[1625054672.46][CONN][RXD] 
[1625054672.46][CONN][RXD] >>> Running case #6: 'Microsecond ticker fire interrupt'...
[1625054672.47][CONN][RXD] >>> 'Microsecond ticker fire interrupt': 1 passed, 0 failed
[1625054672.47][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker fire interrupt}}, queued...
[1625054672.47][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker fire interrupt;1;0}}, queued...
[1625054672.48][CONN][RXD] 
[1625054672.48][CONN][RXD] >>> Running case #7: 'Microsecond ticker overflow test'...
[1625054672.48][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker overflow test}}, queued...
[1625054672.49][CONN][RXD] >>> 'Microsecond ticker overflow test': 1 passed, 0 failed
[1625054672.49][CONN][RXD] 
[1625054672.49][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker overflow test;1;0}}, queued...
[1625054672.50][CONN][RXD] >>> Running case #8: 'Microsecond ticker increment test'...
[1625054672.50][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker increment test}}, queued...
[1625054672.51][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker increment test;1;0}}, queued...
[1625054672.52][CONN][RXD] >>> 'Microsecond ticker increment test': 1 passed, 0 failed
[1625054672.52][CONN][RXD] 
[1625054672.52][CONN][RXD] >>> Running case #9: 'Microsecond ticker speed test'...
[1625054672.53][CONN][INF] found KV pair in stream: {{__testcase_start;Microsecond ticker speed test}}, queued...
[1625054672.53][CONN][INF] found KV pair in stream: {{__testcase_finish;Microsecond ticker speed test;1;0}}, queued...
[1625054672.54][CONN][RXD] >>> 'Microsecond ticker speed test': 1 passed, 0 failed
[1625054672.54][CONN][RXD] 
[1625054672.55][CONN][RXD] >>> Test cases: 9 passed, 0 failed
[1625054672.55][CONN][INF] found KV pair in stream: {{__testcase_summary;9;0}}, queued...
[1625054672.55][CONN][INF] found KV pair in stream: {{end;success}}, queued...
[1625054672.55][CONN][INF] found KV pair in stream: {{__exit;0}}, queued...
[1625054672.55][HTST][INF] __exit(0)
[1625054672.55][HTST][INF] __notify_complete(True)
[1625054672.55][HTST][INF] __exit_event_queue received
[1625054672.55][HTST][INF] test suite run finished after 0.27 sec...
[1625054672.56][CONN][INF] received special event '__host_test_finished' value='True', finishing
[1625054672.56][HTST][INF] CONN exited with code: 0
[1625054672.56][HTST][INF] Some events in queue
[1625054672.56][HTST][INF] stopped consuming events
[1625054672.56][HTST][INF] host test result() call skipped, received: True
[1625054672.56][HTST][INF] calling blocking teardown()
[1625054672.56][HTST][INF] teardown() finished
[1625054672.56][HTST][INF] {{result;success}}
Running ./mbed_hal/echo/cmake_build/sdfx-st-test-echo.bin
[1625054672.96][HTST][INF] host test executor ver. 0.0.15
[1625054672.96][HTST][INF] copy image onto target...
[1625054678.10][MBED][WRN] Target ID not found: Skipping flash check and retry
[1625054678.11][HTST][INF] starting host test process...
[1625054678.12][CONN][INF] starting connection process...
[1625054678.12][CONN][INF] notify event queue about extra 60 sec timeout for serial port pooling
[1625054678.12][CONN][INF] initializing serial port listener... 
[1625054678.12][SERI][INF] serial(port=/dev/ttyACM0, baudrate=115200, read_timeout=0.01, write_timeout=5)
[1625054678.12][HTST][INF] setting timeout to: 60 sec
[1625054678.12][SERI][INF] reset device using 'default' plugin...
[1625054678.23][SERI][INF] waiting 1.00 sec after reset
[1625054679.24][SERI][INF] wait for it...
[1625054679.24][SERI][TXD] mbedmbedmbedmbedmbedmbedmbedmbedmbedmbed
[1625054679.24][CONN][INF] sending up to 2 __sync packets (specified with --sync=2)
[1625054679.24][CONN][INF] sending preamble 'dfa60c77-e962-4527-a62c-03ff9c2fda8f'
[1625054679.24][SERI][TXD] {{__sync;dfa60c77-e962-4527-a62c-03ff9c2fda8f}}
[1625054679.25][CONN][RXD] mbedmbedmbedmbedmbedmbedmbedmbed
[1625054679.26][CONN][INF] found SYNC in stream: {{__sync;dfa60c77-e962-4527-a62c-03ff9c2fda8f}} it is #0 sent, queued...
[1625054679.26][HTST][INF] sync KV found, uuid=dfa60c77-e962-4527-a62c-03ff9c2fda8f, timestamp=1625054679.255200
[1625054679.27][CONN][RXD] >>> Running 1 test cases...
[1625054679.27][CONN][INF] found KV pair in stream: {{__version;1.3.0}}, queued...
[1625054679.27][CONN][INF] found KV pair in stream: {{__timeout;30}}, queued...
[1625054679.27][CONN][INF] found KV pair in stream: {{__host_test_name;device_echo}}, queued...
[1625054679.27][CONN][INF] found KV pair in stream: {{__testcase_count;1}}, queued...
[1625054679.27][HTST][INF] DUT greentea-client version: 1.3.0
[1625054679.27][HTST][INF] setting timeout to: 30 sec
[1625054679.27][HTST][INF] host test class: '<class 'device_echo.Device_Echo'>'
[1625054679.27][HTST][INF] host test setup() call...
[1625054679.27][HTST][INF] CALLBACKs updated
[1625054679.27][HTST][INF] host test detected: device_echo
[1625054679.28][CONN][RXD] 
[1625054679.28][CONN][RXD] >>> Running case #1: 'Echo server: x16'...
[1625054679.28][CONN][INF] found KV pair in stream: {{__testcase_name;Echo server: x16}}, queued...
[1625054679.28][CONN][INF] found KV pair in stream: {{__testcase_start;Echo server: x16}}, queued...
[1625054679.28][CONN][INF] found KV pair in stream: {{echo_count;16}}, queued...
[1625054679.29][SERI][TXD] {{echo_count;16}}
[1625054679.30][CONN][INF] found KV pair in stream: {{echo;abcdefghijklmnopqrstuvwxyzabcdefghi}}, queued...
[1625054679.32][SERI][TXD] {{echo;abcdefghijklmnopqrstuvwxyzabcdefghi}}
[1625054679.33][CONN][INF] found KV pair in stream: {{echo;klmnopqrstuvwxyzabcdefghijklmnopqrs}}, queued...
[1625054679.34][SERI][TXD] {{echo;klmnopqrstuvwxyzabcdefghijklmnopqrs}}
[1625054679.35][CONN][INF] found KV pair in stream: {{echo;uvwxyzabcdefghijklmnopqrstuvwxyzabc}}, queued...
[1625054679.36][SERI][TXD] {{echo;uvwxyzabcdefghijklmnopqrstuvwxyzabc}}
[1625054679.38][CONN][INF] found KV pair in stream: {{echo;efghijklmnopqrstuvwxyzabcdefghijklm}}, queued...
[1625054679.39][SERI][TXD] {{echo;efghijklmnopqrstuvwxyzabcdefghijklm}}
[1625054679.40][CONN][INF] found KV pair in stream: {{echo;opqrstuvwxyzabcdefghijklmnopqrstuvw}}, queued...
[1625054679.41][SERI][TXD] {{echo;opqrstuvwxyzabcdefghijklmnopqrstuvw}}
[1625054679.42][CONN][INF] found KV pair in stream: {{echo;yzabcdefghijklmnopqrstuvwxyzabcdefg}}, queued...
[1625054679.44][SERI][TXD] {{echo;yzabcdefghijklmnopqrstuvwxyzabcdefg}}
[1625054679.45][CONN][INF] found KV pair in stream: {{echo;ijklmnopqrstuvwxyzabcdefghijklmnopq}}, queued...
[1625054679.46][SERI][TXD] {{echo;ijklmnopqrstuvwxyzabcdefghijklmnopq}}
[1625054679.47][CONN][INF] found KV pair in stream: {{echo;stuvwxyzabcdefghijklmnopqrstuvwxyza}}, queued...
[1625054679.49][SERI][TXD] {{echo;stuvwxyzabcdefghijklmnopqrstuvwxyza}}
[1625054679.50][CONN][INF] found KV pair in stream: {{echo;cdefghijklmnopqrstuvwxyzabcdefghijk}}, queued...
[1625054679.51][SERI][TXD] {{echo;cdefghijklmnopqrstuvwxyzabcdefghijk}}
[1625054679.52][CONN][INF] found KV pair in stream: {{echo;mnopqrstuvwxyzabcdefghijklmnopqrstu}}, queued...
[1625054679.54][SERI][TXD] {{echo;mnopqrstuvwxyzabcdefghijklmnopqrstu}}
[1625054679.55][CONN][INF] found KV pair in stream: {{echo;wxyzabcdefghijklmnopqrstuvwxyzabcde}}, queued...
[1625054679.56][SERI][TXD] {{echo;wxyzabcdefghijklmnopqrstuvwxyzabcde}}
[1625054679.57][CONN][INF] found KV pair in stream: {{echo;ghijklmnopqrstuvwxyzabcdefghijklmno}}, queued...
[1625054679.59][SERI][TXD] {{echo;ghijklmnopqrstuvwxyzabcdefghijklmno}}
[1625054679.60][CONN][INF] found KV pair in stream: {{echo;qrstuvwxyzabcdefghijklmnopqrstuvwxy}}, queued...
[1625054679.61][SERI][TXD] {{echo;qrstuvwxyzabcdefghijklmnopqrstuvwxy}}
[1625054679.62][CONN][INF] found KV pair in stream: {{echo;abcdefghijklmnopqrstuvwxyzabcdefghi}}, queued...
[1625054679.63][SERI][TXD] {{echo;abcdefghijklmnopqrstuvwxyzabcdefghi}}
[1625054679.64][CONN][INF] found KV pair in stream: {{echo;klmnopqrstuvwxyzabcdefghijklmnopqrs}}, queued...
[1625054679.66][SERI][TXD] {{echo;klmnopqrstuvwxyzabcdefghijklmnopqrs}}
[1625054679.67][CONN][INF] found KV pair in stream: {{echo;uvwxyzabcdefghijklmnopqrstuvwxyzabc}}, queued...
[1625054679.68][SERI][TXD] {{echo;uvwxyzabcdefghijklmnopqrstuvwxyzabc}}
[1625054679.69][CONN][RXD] >>> 'Echo server: x16': 1 passed, 0 failed
[1625054679.69][CONN][RXD] 
[1625054679.69][CONN][INF] found KV pair in stream: {{__testcase_finish;Echo server: x16;1;0}}, queued...
[1625054679.70][CONN][RXD] >>> Test cases: 1 passed, 0 failed
[1625054679.71][CONN][INF] found KV pair in stream: {{__testcase_summary;1;0}}, queued...
[1625054679.71][CONN][INF] found KV pair in stream: {{end;success}}, queued...
[1625054679.71][CONN][INF] found KV pair in stream: {{__exit;0}}, queued...
[1625054679.71][HTST][INF] __exit(0)
[1625054679.71][HTST][INF] __notify_complete(True)
[1625054679.71][HTST][INF] __exit_event_queue received
[1625054679.71][HTST][INF] test suite run finished after 0.44 sec...
[1625054679.72][CONN][INF] received special event '__host_test_finished' value='True', finishing
[1625054679.72][HTST][INF] CONN exited with code: 0
[1625054679.72][HTST][INF] Some events in queue
[1625054679.72][HTST][INF] stopped consuming events
[1625054679.72][HTST][INF] host test result() call skipped, received: True
[1625054679.72][HTST][INF] calling blocking teardown()
[1625054679.72][HTST][INF] teardown() finished
[1625054679.72][HTST][INF] {{result;success}}

RUN SUMMARY:
./mbed_hal/us_ticker/cmake_build/sdfx-st-test-us_ticker.bin
./mbed_hal/us_ticker_lp_ticker_frequency/cmake_build/sdfx-st-test-us_ticker_lp_ticker_frequency.bin
./mbed_hal/us_ticker_lp_ticker_common/cmake_build/sdfx-st-test-us_ticker_lp_ticker_common.bin
./mbed_hal/echo/cmake_build/sdfx-st-test-echo.bin
```

</details>

Obviously, I ran the `build_all.sh` before `run_all.sh`. :wink:

**Review by MarceloSalazar — APPROVED** [review]

**Pull request merged**